### ISSUE TITLE
Platform Requirements for 17.x

### ISSUE THREAD

**Issue by AshCripps** [nodejs/build#2740]
Its that time of year again

## What we build and test on

From https://github.com/nodejs/node/blob/master/BUILDING.md#official-binary-platforms-and-toolchains

Binary package | Platform and Toolchain
-- | --
aix-ppc64 | AIX 7.2 TL04 on PPC64BE with GCC 8
darwin-x64 | macOS 10.15, Xcode Command Line Tools 11 with -mmacosx-version-min=10.13
darwin-arm64 (and .pkg) | macOS 11 (arm64), Xcode Command Line Tools 12 with -mmacosx-version-min=10.13
linux-arm64 | CentOS 7 with devtoolset-8 / GCC 8 8
linux-armv7l | Cross-compiled on Ubuntu 18.04 x64 with custom GCC toolchain
linux-ppc64le | CentOS 7 with devtoolset-8 / GCC 8 8
linux-s390x | RHEL 7 with devtoolset-8 / GCC 8 8
linux-x64 | CentOS 7 with devtoolset-8 / GCC 8 8
win-x64 and win-x86 | Windows 2012 R2 (x64) with Visual Studio 2019

## Compiler levels

Operating System | Compiler Versions
-- | --
Linux | GCC >= 8.3
Windows | Visual Studio >= 2019 with the Windows 10 SDK on a 64-bit host
macOS | Xcode >= 11 (Apple LLVM >= 11)

## Actions

I dont think theres much to do this time around? cc @nodejs/build

**Comment by nschonni** [comment]
Does it make sense to raise Alpine/musl from Experimental to Tier 2 for the new release line?

**Comment by nschonni** [comment]
Right, I think it's still mostly just @rvagg and @richardlau that do the care and feeding for those on the Unofficial Builds still.

**Comment by rvagg** [comment]
The @nodejs/docker folks are pretty on-the-ball wrt Alpine builds, although I'm not sure there's anyone on there that wants to be on the hook to support core when things break. We do already have Alpine in our CI mix so breakage is already something we look for with every commit and it's been pretty stable, so I think it might even already qualify for Tier 2:

> Tier 2: These platforms represent smaller segments of the Node.js user base. The Node.js Build Working Group maintains infrastructure for full test coverage. Test failures on tier 2 platforms will block releases. Infrastructure issues may delay the release of binaries for these platforms.

@nodejs/docker do we have any people willing to be put on a `@nodejs/musl` group to be go-to if/when weird musl things break that nobody else has expertise or setups to debug?
